client_test.go

```diff
@@ -920,6 +920,7 @@ func testPipelineClientDoConcurrent(t *testing.T, concurrency int, maxBatchDelay
 
 	s := &Server{
 		Handler: func(ctx *RequestCtx) {
+			ctx.SetStatusCode(200)
```

Comment:
Sorry, this is not what I had in mind. If this is required for the change to work then it will break for all users that aren't doing this at the moment. That in unacceptable. I think 200 should be set as the default status code if nothing has been set once the handler returns.

Reply:
Ok, i proposed second solution (PR #1198). Maybe it will be softer :)